### ISSUE TITLE
How to calculate the F1 and accuracy from the predicted results as per paper?

### ISSUE THREAD

**Issue by monajalal** [jbohnslav/deepethogram#27]
Hi Jim,

In your paper, there is this photo and we would like to know what script you have used for calculating the accuracy and F1 score.

<img width="701" alt="Screen Shot 2020-11-30 at 5 16 25 PM" src="https://user-images.githubusercontent.com/1892917/100676858-b81c1c80-3337-11eb-9e97-d9a6019026cf.png">

**Comment by jbohnslav** [comment]
Each the rows has the accuracy / f1 calculated independently. For F1, sklearn: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html. For accuracy, `np.mean(labels == predictions)`
